lib/chefdepartie.rb

```diff
@@ -7,3 +7,5 @@
 require 'chefdepartie/cookbook'
 require 'chefdepartie/databag'
 require 'chefdepartie/cache'
+
+Chefdepartie.run(background: true, config: 'config.rb', cache: nil)
```

Comment:
Yes it was done for testing.  Also was following the README, which doesn't work without this line...  i.e. `bundle exec ruby lib/chefdepartie.rb` simply returns without it.